showcase/app/controllers/application.ts

```diff
@@ -7,18 +7,23 @@ import Controller from '@ember/controller';
 import { inject as service } from '@ember/service';
 import { tracked } from '@glimmer/tracking';
 
+import type RouterService from '@ember/routing/router-service';
+import type Owner from '@ember/owner';
+
 export default class ApplicationController extends Controller {
-  @service router;
+  @service declare router: RouterService;
```

Comment:
```suggestion
  @service declare readonly router: RouterService;
```
[Suggestion] In the converted utilities file we add `readonly` to the `router`. We could do the same here.